makefile.in

```diff
@@ -44,11 +44,11 @@ quickclean                    : $(CLEANPROGS) quickclean-binutils quickclean-gcc
 $(PROGS)                      :
 	cd src/$@ && $(MAKE)
 $(INSTALLPROGS)               :
-	cd src/$(patsubst install-%,%,$@) && $(MAKE) install
+	cd src/$(patsubst install-%,%,$@) && $(MAKE) DESTDIR=$(DESTDIR) install
```

Comment:
`DESTDIR` should not be forwarded manually to sub-makes. make will forward it automatically.

Reply:
I tested it, and removing DESTDIR from this line makes it so $(DESTDIR) is empty in sub-makes.

Reply:
How are you setting `DESTDIR` in your test? During packaging it would be set at the command line when the top make is invoked, variables set this way are always forwarded automatically.

Reply:
Ok, I tested both by invoking `configure` and `make` directly in the source directory and from the PKGBUILD and it works. You're right, I probably messed up something inthe PKGBUILD when I was testing it. Probably an unclean source folder. Done.

Reply:
Some manual forwarding remains in the `install-strip` targets.

Reply:
Sorry about that. Now it's done.